leo-finalize.md

```diff
@@ -0,0 +1,80 @@
+
+### Leo Finalize Update
```

Comment:
Make sure that the Headers follow the H1, H2, H3, structure. So that the main titles are H1 (`# Main title`), and secondary titles in the same page are declared as H2 (`## Secondary Title`)